phantazm-zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/gun/target/TargetFinder.java

```diff
@@ -10,17 +10,40 @@
 import java.util.Objects;
 import java.util.UUID;
 
+/**
+ * Finds targets for a shot.
+ * While entity finders find possible entity candidates, implementations of this class
+ * should find actual targets: both regular shots and headshots.
```

Comment:
should perhaps specify that "target" means "entity that will be hit by whatever shot" and entity candidates are just the entities that are checked to see if they even should become targets